doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -3,6 +3,8 @@ RaySGD Pytorch
 
 .. warning:: This is still an experimental API and is subject to change in the near future.
 
+.. tip:: Help take a 1 minute `User Survey for the Ray Distributed Training library <https://forms.gle/26EMwdahdgm7Lscy9>`_!
```

Comment:
How about "tip:: Help us make Ray distributed Training better; take this 1 minute user survey..."